src/components/loader/loader.js

```diff
@@ -48,7 +48,21 @@ export default class Loader extends BaseObject {
   constructor(externalPlugins, playerId) {
     super()
     this.playerId = playerId
-    this.playbackPlugins = [HLSVideoPlayback, HTML5VideoPlayback, HTML5AudioPlayback, FlashVideoPlayback, FlasHLSVideoPlayback, HTMLImgPlayback, NoOp]
+    this.playbackPlugins = [
+      HTML5VideoPlayback,
+      HTML5AudioPlayback,
+      HTMLImgPlayback,
+      NoOp
+    ]
+
+    if (!PLAIN_HTML5_ONLY === 'yes') {
```

Comment:
Why don't we use true/false here? This way the `if` statement would be much simpler:
```
if (PLAIN_HTML5_ONLY)
```

Reply:
yes, but it's tricky i think... :) we are getting this from the environment, so per se it is a always going to be a `string`. Shell rules are that we have to give it some value: env var can't be empty i think, that's same as not existing, so it will simply not exist in `process.env`. And when we do Boolean(someString) it will be `true` except if the string is zero length. What we could do in the webpack file would be: `Boolean(process.env.CLAPPR_...)` which will result to `true` for any value of the env var. Or we do `(process.env.CLAPPR_... === 'yes')` or `(process.env.CLAPPR_... === 'true')`, which will force users to certain values. I think that's better. What do you prefer? 

I guess the deeper question really is how to deal with env vars in your build conf.

Reply:
I think we can hold off a little bit the generic solution and reach a middle ground for now. 

Currently, we're trying to have a PLAIN_HTML5_ONLY build and the default one, with all plugins. So, we just need to enable the PLAIN_HTML5_ONLY flag. In the `Loader` code, I believe it's better to have it as a boolean value since we don't need to have the possible string values of the environment variable there.

As for the `process.env.CLAPPR_...` variable, we could either use true/false for it, having the empty/non-existent/false values to disable it and using whatever else for true, or we can force it to have a yes/no value. I think the first option would be better, since the DefinePlugin allows boolean values, AFAIK.

This way, instead of having:
```
new webpack.DefinePlugin({
  VERSION: JSON.stringify(require('./package.json').version)
  PLAIN_HTML5_ONLY: JSON.stringify(process.env.CLAPPR_PLAIN_HTML5_ONLY)
})
```
we could have:
```
new webpack.DefinePlugin({
  VERSION: JSON.stringify(require('./package.json').version)
  PLAIN_HTML5_ONLY: !!process.env.CLAPPR_PLAIN_HTML5_ONLY
})
```

Reply:
Just to make things easier, we could also add scripts that set this environment var automatically, like:
```
"scripts": {
    "build": "webpack --config webpack.dev.js --progress",
    "build:plainhtml5": "CLAPPR_PLAIN_HTML5_ONLY=true webpack --config webpack.dev.js --progress",
   ...
}
```

Reply:
OK, totally agreeing! :+1: